source/acquaman/AMScanConfiguration.h

```diff
@@ -168,6 +168,12 @@ class AMScanConfiguration : public AMDbObject
 	virtual QString technique() const;
 
 	/*!
+	  * Returns whether this is an XAS technique scan
+	  */
+
+	bool isXASScan() const;
```

Comment:
@dretrex , do you want me to remove all the "Scan" in the technique()? So far, we have technique() return "XAS Scan", "2D Scan", "3D Scan", "Line Scan", "Energy Scan", "Timed Scan", "Timed Line Scan", "Generic Continuous Scan", "Generic Step Scan", and "X-Ray Fluorescene Scan".